submissions/sprint_a/app.py

```diff
@@ -0,0 +1,12 @@
+from sprint_a import app
+from flask import jsonify
```

Comment:
Check the order of your imports: https://www.python.org/dev/peps/pep-0008/#imports

Imports should be grouped in the following order:
Standard library imports.
Related third party imports.
Local application/library specific imports.
You should put a blank line between each group of imports.